[2018-08-30T13:36:27.644] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T13:36:29.710] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T13:37:38.147] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T13:37:38.572] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T17:05:29.104] [ERROR] errorLog - TypeError: Cannot read property 'home' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:10:75
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:10:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-30T17:05:29.406] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T17:08:04.253] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T17:08:04.262] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T17:08:04.435] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T17:09:00.248] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T17:09:00.429] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T17:11:19.027] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T17:11:19.213] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:14:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:122:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:05:03.908] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:09:33.357] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T18:09:36.580] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T18:25:08.831] [ERROR] errorLog - { Error: ENOENT: no such file or directory, stat 'G:\node\node-service-frame\dev\index.html'
  errno: -4058,
  code: 'ENOENT',
  syscall: 'stat',
  path: 'G:\\node\\node-service-frame\\dev\\index.html' }
[2018-08-30T18:27:38.400] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:27:38.410] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:27:49.545] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:10:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:28:35.796] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:13:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:28:53.069] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:13:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:29:03.985] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:13:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:29:43.714] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:13:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-30T18:29:53.785] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:12:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:120:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at goNext (G:\node\node-service-frame\node_modules\webpack-dev-middleware\lib\middleware.js:16:16)
    at processRequest (G:\node\node-service-frame\node_modules\webpack-dev-middleware\lib\middleware.js:64:26)
    at cbs.forEach (G:\node\node-service-frame\node_modules\webpack-dev-middleware\lib\context.js:52:9)
